### PR TITLE
Updating tekton file for kserv 210

### PR DESCRIPTION
**Pull request by moulalis**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Checklist**:

- [ ] Have you added unit/e2e tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

**Release note**:

```release-note

```

### CODE REVIEW

File: .tekton/kserve-controller-210-pull-request.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
-      == "rhoai-2.10" && (!".tekton/*.*".pathChanged() || ".tekton/kserve-controller-210-pull-request.yaml".pathChanged()) && !"config/overlays/odh/params.env".pathChanged()
+      == "rhoai-2.10" && ( !".tekton/**".pathChanged() || ".tekton/kserve-controller-210-pull-request.yaml".pathChanged() ) && !"config/overlays/odh/params.env".pathChanged()
```
Comment (riprasad) [inline .tekton/kserve-controller-210-pull-request.yaml:11]: @moulalis Can you use the same format I used in the main branch. That's more readable

Comment (moulalis) [reply .tekton/kserve-controller-210-pull-request.yaml:11]: > @moulalis Can you use the same format I used in the main branch. That's more readable

I copied it from main branch file

Comment (moulalis) [reply .tekton/kserve-controller-210-pull-request.yaml]: > @moulalis Can you use the same format I used in the main branch. That's more readable

@riprasad I've updated the changes as expected.

**Comment by riprasad** [comment]
LGTM

**Pull request merged**